app/views/layouts/_sidebar.html.erb

```diff
@@ -40,10 +40,12 @@
         </ul>
         <p class="Docs__nav__section-subheading">Elastic CI Stack for AWS</p>
         <ul class="Docs__nav__sub-nav">
-          <%= sidebar_link_to "Running Elastic CI Stack", 'agent/v3/elastic-ci-aws' %>
-          <%= sidebar_link_to "Template Parameters", 'agent/v3/elastic-ci-aws/parameters' %>
+          <%= sidebar_link_to "Overview", 'agent/v3/elastic-ci-aws/elastic-ci-stack-overview' %>
+          <%= sidebar_link_to "Elastic CI Stack for Linux", 'agent/v3/elastic-ci-aws' %>
```

Comment:
This isn't just Linux, it's either just Elastic CI Stack, or it's Linux and Windows. Not quite sure which is the least worst option here.

Reply:
My thoughts exactly :D "Not quite sure which is the least worst option"
I'll go with "just Elastic CI Stack".